bliss/utils.py

```diff
@@ -139,3 +140,42 @@ def forward(self, mean_z, logscale_z):
         if self.minscale is not None:
             logscale_z = torch.log(self.minscale + (1 - self.minscale) * F.softplus(logscale_z))
         return Normal(mean_z, logscale_z.exp())
+
+
+# adapt from torchvision.utils.make_grid
+@torch.no_grad()
+def make_grid(
```

Comment:
Can you add documentation just explaining the output of this function (or how it is different from `torchvision.utils.make_grid`, anything else that is important?)

Reply:
I think @ZachariahPang can better answer this question